src/main/java/com/stripe/model/StripeSearchResult.java

```diff
@@ -18,6 +18,9 @@
   @Getter(onMethod_ = {@Override})
   List<T> data;
 
+  @Getter(onMethod_ = {@Override})
+  Integer totalCount;
```

Comment:
I believe this should be a `Long` and not `Integer` (https://github.com/stripe/stripe-java/commit/b656eac45ab1e843b508dc5dbb080e916441b0ae for what it used to be in lists).

Also we should add a comment indicating that it's only present if the user explicitly expands `total_count`.